src/array.rs

```diff
@@ -569,6 +569,18 @@ pub trait ArrayValue:
     fn format_sep() -> &'static str {
         " "
     }
+    /// Row delimiters that can be read by the interpreter
+    fn debug_delims() -> (&'static str, &'static str) {
+        ("[", "]")
+    }
+    /// String representation that can be read by the interpreter
+    fn debug_string(&self, _depth: usize, _prefix: bool) -> String {
+        self.to_string()
+    }
+    /// Separator for repr
+    fn debug_sep() -> &'static str {
+        Self::format_sep()
+    }
```

Comment:
I'd like to separate these functions from the main `ArrayValue` trait.
Can you make a super trait, i.e. `trait DebugArrayValue: ArrayValue` please?
Put those implementaiton at the bottom of the file.

Reply:
done